tests/Helper/MockAuthentication.php

```diff
@@ -17,7 +17,7 @@
 use OpenCFP\Domain\Services\NotAuthenticatedException;
 use OpenCFP\Infrastructure\Auth\UserInterface;
 
-final class MockableAuthenticator implements Authentication
```

Comment:
The actual implementation of the `Authentication` interface is called [`SentinelAuthentication`](https://github.com/opencfp/opencfp/blob/844710aa1ea60a761696007741c7d703fdf89ecf/classes/Infrastructure/Auth/SentinelAuthentication.php), so `Authentication` here makes more sense, too.

Also, it's not actually `Mockable`, as the class itself is `final`, so `MockAuthentication` would be more appropriate, too, I think.